site/src/components/site-header.md

```diff
@@ -0,0 +1,499 @@
+		</svg>
+	</a>
+
+	<!-- Divider -->
+	<div class="site-header-divider"></div>
+
+	<!-- Brand -->
+	<a href="~/src/index.md" class="site-header-brand">
+		<span>Brand</span>
+	</a>
+
+	<!-- Navigation -->
+	<nav class="site-header-nav">
+		<ul class="display-flex gap-xxs">
+			<li>
+				<a href="#!" class="site-header-button">
+                    <span>Link 1</span>
+                </a>
+			</li>
+			<li>
+				<a href="#!" class="site-header-button">
+                    <span>Link 2</span>
+                </a>
+			</li>
+			<li>
+				<a href="#!" class="site-header-button">
+                    <span>Link 3</span>
+                </a>
+			</li>
+		</ul>
+	</nav>
+</div>
+```
+
+### Logo
+
+There is an atomic available to position the logo right in the middle of header - `site-header-logo-centered`.
+
+```html-no-indent
+<div class="site-header justify-content-space-between">
+	<!-- Brand -->
+	<a href="~/src/index.md" class="site-header-brand">
+		<span>Brand</span>
+	</a>
+
+    <!-- Microsoft logo -->
+	<a href="https://www.microsoft.com" aria-label="Microsoft" class="site-header-logo-centered">
+		<svg
+			class="site-header-logo"
+			aria-hidden="true"
+			viewBox="0 0 25 25"
+			fill="none"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			<path d="M11.5216 0.5H0V11.9067H11.5216V0.5Z" fill="#f25022" />
+			<path d="M24.2418 0.5H12.7202V11.9067H24.2418V0.5Z" fill="#7fba00" />
+			<path d="M11.5216 13.0933H0V24.5H11.5216V13.0933Z" fill="#00a4ef" />
+			<path d="M24.2418 13.0933H12.7202V24.5H24.2418V13.0933Z" fill="#ffb900" />
+		</svg>
+	</a>
+
+	<a href="#!" class="link-button font-size-sm">
+		<span>Link</span>
+	</a>
+</div>
+```
+
+### Panel
+
+There is a Panel sub-component available to use within site header. Panel provides additional space to hold more content. Panel is composed of the following elements:
+
+| Component                            | Description                                                                                          |
+| ------------------------------------ | ---------------------------------------------------------------------------------------------------- |
+| `site-header-panel`                  | The main container for the site header panel.                                                        |
+| `site-header-panel-featured-content` | Featured section in the panel. Located on the right side and it's recommended to hold up to 2 cards. |
+| `site-header-panel-content`          | Container that holds the main content. It should be either the set of links or cards.                |
+| `site-header-panel-links`            | Section with set of links. Recommended to have up to 21 links.                                       |
+| `site-header-panel-cards`            | Section with set of cards. Recommended to have up to 9 cards.                                        |
+
+Below are the examples of both content options - with a set of links and a set of cards:
```

Comment:
Below are the examples of both content options, with a set of links and a set of cards: